### ISSUE TITLE
Create a new release including recent changes (esp. dependency updates)

### ISSUE THREAD

**Issue by laodouya** [ultralytics/yolov5#13637]
### Search before asking

- [x] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar feature requests.

### Description

Hi Ultralytics team,

Thank you for your continuous work on YOLOv5. The latest official GitHub release remains **v7.0** from November 22, 2022, but the `main` branch continues to receive active updates regularly.

Could you please cut a **new release** (e.g. `v7.x` or `v8.0`) that bundles recent changes? It would be really helpful if this release note also includes:

- A clear summary of dependency/environment changes since the last release (e.g. updated `requirements.txt`, Python 3.8+ / PyTorch 1.8+ requirements)
- A corresponding tagged **Docker image** on DockerHub aligned to the new GitHub release, so users can pull `ultralytics/yolov5:<tag>`

Although users can pull `ultralytics/yolov5:latest`, it’s safer for CI and production pipelines to lock to a stable, documented tag instead of floating `latest`, which can break unpredictably when `main` moves forward.
Happy to help with reviewing changelog or testing if that’s useful!

Best regards,
laodouya

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @laodouya, thank you for your interest in YOLOv5 🚀 and for your detailed feedback! This is an automated response to help get you started—an Ultralytics engineer will review your request and assist you soon.

Please check out our ⭐️ [Tutorials](https://docs.ultralytics.com/yolov5/) for quickstart guides, including [Custom Data Training](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data/) and advanced topics like [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution/).

If this is a 🐛 Bug Report, please provide a **minimum reproducible example** to help us debug if you haven’t already.

If your request involves custom training or environment details, please share as much info as possible, such as dataset samples, logs, and confirm you’re following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

## Requirements
[**Python>=3.8.0**](https://www.python.org/) with all [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt) installed including [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/). To get started:
```bash
git clone https://github.com/ultralytics/yolov5  # clone
cd yolov5
pip install -r requirements.txt  # install
```
## Environments
YOLOv5 can be run in any of these up-to-date verified environments (with all dependencies such as [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):
- **Notebooks** with free GPU: <a href="https://bit.ly/yolov5-paperspace-notebook"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>
## Status
<a href="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml"><img src="https://github.com/ultralytics/yolov5/actions/workflows/ci-testing.yml/badge.svg" alt="YOLOv5 CI"></a>
If this badge is green, all [YOLOv5 GitHub Actions](https://github.com/ultralytics/yolov5/actions) Continuous Integration (CI) tests are currently passing. CI tests verify correct operation of YOLOv5 [training](https://github.com/ultralytics/yolov5/blob/master/train.py), [validation](https://github.com/ultralytics/yolov5/blob/master/val.py), [inference](https://github.com/ultralytics/yolov5/blob/master/detect.py), [export](https://github.com/ultralytics/yolov5/blob/master/export.py) and [benchmarks](https://github.com/ultralytics/yolov5/blob/master/benchmarks.py) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

Thank you again for your suggestion on new releases and Docker tags—your feedback helps us improve! An Ultralytics engineer will follow up with you soon.

**Comment by pderrenger** [comment]
Thanks for the thoughtful request @laodouya! You're absolutely right that having tagged releases is important for production stability and CI pipelines. The [YOLOv5 v7.0 release](https://www.ultralytics.com/blog/introducing-instance-segmentation-in-yolov5-v7-0) introduced instance segmentation models and there have been numerous improvements since then.

I'll pass your request along to the development team for consideration. Release timing involves coordination across multiple factors including feature completeness, testing, and documentation updates. Your offer to help with changelog review and testing is much appreciated - community contributions like yours are invaluable for maintaining quality releases.

In the meantime, you might consider pinning to a specific commit hash in your CI/production workflows for the same stability benefits while we work toward the next tagged release.
